src/main/kotlin/onm/builder/HouseBuilder.kt

```diff
@@ -0,0 +1,62 @@
+package onm.builder
+
+import onm.configuration.RoomType
+import onm.events.EventHandler
+import onm.events.IEventHandler
+import onm.house.devices.Fridge
+import onm.house.devices.Washer
+import onm.house.places.RoomBuilder
+import java.util.*
+
+/**
+ * Class representing framework builder for devices and rooms in a smart house.
+ */
+class HouseBuilder(){
+
+
+    /**
+     * IEvent handler dealing with all raised events from i.e., devices
+     */
+    val eventHandler : IEventHandler = EventHandler()
```

Comment:
this should be injected via constructor but it could be done in future, just leaving this comment here